docs/guides/Payments/payments-configuration-guides/setting-up-merchant-id-in-apple-pay.md

```diff
@@ -127,10 +127,10 @@ Thereafter, to upload the`.txt` file by API, follow these instructions:
 
 Then, start a Postman session. You perform a call to VTEX's CDN using the POST method:
 
-1. Configure the route **POST** `https://{{yourdomain}}/.well-known/raw/apple-developer-merchantid-domain-association.txt`.
+1. Configure the route **POST** `https://{{yourdomainhere}}/.well-known/raw/apple-developer-merchantid-domain-association.txt?persistent=true`.
 2. Ensure that all contents of the .txt files are in:
```

Comment:
No Postman, não será upload de um txt file, né? É para copiar o conteúdo do txt e colar no request body como json raw, certo? Acho que vale deixar isso mais explicado aqui.

Reply:
No passo anterior estamos descrevendo a necessidade de adicionar copiar o conteúdo do .txt e depois mostramos em qual formato deve estar ao realizar a chamada.

"2. Open the .txt file and copy all the content.

Then, start a Postman session. You perform a call to VTEX's CDN using the POST method:

1. Configure the route **POST** `https://{{yourdomainhere}}/.well-known/raw/apple-developer-merchantid-domain-association.txt?persistent=true`

2. Ensure that all contents of the .txt files are in:
- Quotations marks
- JSON format, inside a "token_content" body
- Without any line breaks.
"

Reply:
Achei a parte de copiar clara, mas aqui acaba não ficando tão claro o que fazer com o txt, mesmo ele estando no formato certo. Mudaria o passo 2 para algo tipo:

```
2. Paste the content of the .txt file in the request body. Ensure that all contents of the file are:
   - Inside quotations marks.
   - In JSON format, as in `"{token_content}"`.
   - Without any line breaks.
```

Reply:
Corrigido.